File: specs/ts/line.ts
import { Spec } from '@uwdata/mosaic-spec';

export const spec : Spec = {
  "data": {
    "aapl": {
      "file": "data/stocks.parquet",
      "where": "Symbol = 'AAPL'"
    }
  },
  "plot": [
    {
      "mark": "lineY",
      "data": {
        "from": "aapl"
      },
      "x": "Date",
      "y": "Close"
    }
  ],
  "width": 680,
  "height": 200
};

File: specs/python/line.py
import vgplot as vg

aapl = vg.parquet("data/stocks.parquet", where="Symbol = 'AAPL'")

view = vg.plot(vg.line_y(aapl, x="Date", y="Close"), vg.width(680), vg.height(200))

view
